modules/aws-emr-iam/main.tf

```diff
@@ -55,30 +57,57 @@ data "aws_iam_policy_document" "emr_service_policy_1" {
     resources = ["*"]
   }
 
+  ## ABAC OK - copied from docs
   statement {
+    sid    = "ManageTagsOnEMRTaggedResources"
     effect = "Allow"
     actions = [
-      "iam:GetRole",
-      "iam:GetRolePolicy",
-      "iam:ListInstanceProfiles",
-      "iam:ListRolePolicies",
-      "iam:PassRole"
+      "ec2:CreateTags",
+      "ec2:DeleteTags"
     ]
-    resources = ["*"]
+    resources = [
+      "arn:aws:ec2:*:*:instance/*",
+      "arn:aws:ec2:*:*:volume/*",
+      "arn:aws:ec2:*:*:network-interface/*",
+      "arn:aws:ec2:*:*:launch-template/*"
```

Comment:
Replace `"arn:aws:ec2:*:*:<RESOURCE_TYPE>/*"` with `"arn:${var.arn_partition}:ec2:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:<RESOURCE_TYPE>/*"`

Reply:
Addressed with the refactoring